qds_sdk/cluster.py

```diff
@@ -286,7 +330,9 @@ def _parse_create_update(cls, args, action):
                                  default=None,
                                  help="don't encrypt the ephemeral drives on" +
                                       " the instance",)
-        security_group.add_argument("--customer-ssh-key",
+
+        customer_ssh_key_parameter = "--ssh-public-key" if (api_version >= 1.3) else "--customer-ssh-key"
```

Comment:
Why is this different based on api version? We can keep the command line flag same even if the API argument is different.